gtests/net/packetdrill/parser.y

```diff
@@ -1875,7 +1875,7 @@ add_addr_ip
 | ADDR '[' WORD '=' INET6_ADDR '(' STRING ')' ']' {
 	struct endpoint endpoint;
 	memset(&endpoint, 0, sizeof(endpoint));
-	struct ip_address ip_formatted = ipv6_parse($5);
+	struct ip_address ip_formatted = ipv6_parse($7);
```

Comment:
may you add a small explanation in the commit message if you don't mind?

It looks good to me when looking at what is done in IPv4. I guess this was not validated before.

Reply:
thx!

> I think it's just a typo, and probably it never worked since the beginning

That's what I understood when reading the patch but it was "strange". Strange the code was there but never validated (but maybe not that strange here :) )